apps/search/scripts/migrations/update-webhooks.ts

```diff
@@ -43,7 +43,7 @@ export const updateWebhooksScript = async ({
 
       const baseUrl = new URL(targetUrl).origin;
 
-      return appWebhooks.map((w) => ({ ...w.getWebhookManifest(baseUrl), enabled }));
+      return appWebhooks.map((w) => ({ ...w.getWebhookManifest(baseUrl), isActive }));
```

Comment:
How I understand it, is that you only changed variable name, but the uderlying logic is the same. Where exactly you did this:

> Active webhooks are no longer disabling themselves when running migrations. This means when some webhook is active, migration will turn on all of them.

?

Reply:
- Webhooks that belong to this app are disabled by default: https://github.com/saleor/apps/blob/main/apps/search/src/webhooks/definitions/product-created.ts#L13 and the flag resposible for it is called "isActive"
- Despite the `appDetails` is being loaded and current configuration of webhooks is being checked: https://github.com/saleor/apps/pull/1559/files/202590cfbf8408744ff4b5962f8b185bafed4399#diff-44df2f0686ba642deefd1ceaa6ddd7fec5dcd11f616df6f614e732247d275f59R35-R36 we set fag "enabled" instead of "isActive"
- `getWebhookManifest`  returns for me defaults, the values I have within the definitions, so we set "isActive" to false again ("enabled" is not even used)

Reply:
Now, when I have some webhooks already enabled, I check that against the appDetails and if some are enabled, i switch isActive to true for all my webhooks

Reply:
But I dont know how this answers by question? The code I see is not changing the logic

Reply:
![CleanShot 2024-09-09 at 08 36 46](https://github.com/user-attachments/assets/50698f27-dc1f-4dfd-9489-c2e0e9bf5a97)

you changed variable name

Reply:
as I said above - the vaibalbe name that we need to set is `IsAvtive`, not `enabled`, and the code sets sets this to `false` each time we run this migration because that's the [default value](https://github.com/saleor/apps/blob/main/apps/search/src/webhooks/definitions/product-created.ts#L13), it's just not being set properly.

Reply:
discussed 👍